module1/lessons/hashes.md

```diff
@@ -77,14 +93,39 @@ Ex:
       dogs: 9
  }
 ```  
+<section class="dropdown">
+
+### Javascript <h3>
+```js
+var toppings = {
+    pepperoni: 5,
+    sausage: 3,
+    pineapple: 500
+};
+
+var pets = {
+    cats: 6,
+    dogs: 9
+};
+```
+
+</section>
 
 
 ### What data type can a key be?
   
 Keys of a hash can be any data type. 
-Often, we see them as symbols.
-A symbol is an immutable data-type in Ruby. They will start with a `:` (colon).
-Keys can also be seen commonly as strings, however they can be integers as well.
+Often, we see them as symbols (in Ruby) or strings (in Javascript).However, they can be integers as well.
```

Comment:
Add a space after the period, before the word `However`

Reply:
Fixed it here [cb03455](https://github.com/turingschool/curriculum-site/pull/59/commits/cb03455e135e37994804f34ee67328c71214bbd5)